packages/terra-clinical-alert/src/Alert.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconRequired from 'terra-icon/lib/icon/IconRequired';
+import IconDiamond from 'terra-icon/lib/icon/IconDiamond';
+import BaseAlert from 'terra-alert';
+import classNames from 'classnames';
```

Comment:
This needs to be updated to 'classnames/bind' to make the the classNames style-bound.